paintBrush.js

```diff
@@ -14,15 +32,26 @@ if (canvas) {
     canvas.addEventListener('mouseleave', stopPainting);
 }
 
+/** Function to disconnect limes when not painting, paint where mouse is when clicking down
+ * Dependant on mode set, use black for brush and white for eraser
+ *
+ * @param event
+ */
 function drawLine(event) {
-    ctx.lineTo(event.offsetX, event.offsetY)
     if (!painting) {
         // When not painting, begin a new path
         ctx.beginPath();
         ctx.moveTo(event.offsetX, event.offsetY);
     } else {
         // When painting, draw a line to...
         ctx.lineTo(event.offsetX, event.offsetY)
+        // If the mode is set to brush, draw in black
+        if (mode === 'brush') {
+            ctx.strokeStyle = '#000000';
+            // if the mode is set to eraser, draw white lines
+        } else if (mode === 'eraser') {
+            ctx.strokeStyle = '#FFFFFF';
```

Comment:
When in erase mode, could you also increase the width of the stroke? I find it quite hard to actually erase my lines accurately when the eraser is the exact same size. I reckon making it so the eraser is double the width would do the trick